tests/e2e/test_project.py

```diff
@@ -11,20 +11,22 @@ def test_project_init(helper: Helper) -> None:
     path = pathlib.Path(f"./name-of-the-project")
     assert not path.exists()
     try:
-        helper.run_cli(["project", "init", "-q"])
+        helper.run_cli(["project", "init", "--no-input"])
         assert path.exists()
         assert path.is_dir()
     finally:
-        shutil.rmtree(path)
+        if path.exists():
```

Comment:
It still does not look right. If `path.exists()` can be false then `assert path.exists()` will fail. If remove asserts, we will loss the way to check that `project init` works.

Reply:
What I observe now is that `project init` fails with exception on Windows. I'm investigating that first.